hw/ip/usbdev/lint/usbdev.waiver

```diff
@@ -3,3 +3,10 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 # waiver file for usbdev
+
+waive -rules MULTI_PROC_ASSIGN -location {prim_generic_ram_2p.sv} -regexp {Assignment to 'mem' from more than one block} \
+      -comment "That is the nature of a dual-port memory: both write ports can access the same storage simultaneously"
+waive -rules HIER_NET_NOT_READ -regexp {Net 'reg_wdata\[[0-9]+:[0-9]+\]' is not read from in module 'usbdev_reg_top'} \
+      -comment "The TL-UL socket outputs the full 32 bits of write data to both the register interface and the SRAM window, but only the SRAM window uses the full width."
+waive -rules HIER_READ -regexp {Signal 'reg_wdata\[[0-9]+:[0-9]+\]' is not read from in module 'usbdev_reg_top'} \
```

Comment:
I believe this should be `NOT_READ` instead of  `HIER_READ`

Reply:
Correct, thanks!